src/WebResponses.cpp

```diff
@@ -260,11 +255,29 @@ String AsyncWebServerResponse::_assembleHead(uint8_t version) {
 #endif
   out.concat(buf);
 
+  bool _sendAcceptsRange = true,
+    _sendTransferEncoding = true,
+    _sendContentType = true;
+
+  for (const auto& header : _headers) {
+    const char* headerName = header.name().c_str();
+    _sendAcceptsRange &= (strcasecmp(headerName, T_Accept_Ranges) == 0);
+    _sendTransferEncoding &= (strcasecmp(headerName, T_Transfer_Encoding) == 0);
```

Comment:
@mathieucarbou Would this suffice? Would appreciate your feedback!

Reply:
You can optimise the code using == to compare before using strcasecmp : if am not mistaken `T_Accept_Ranges    == "Accept-Ranges"` should be true because the compiler puts the strings at the same address, so if user passes a hardcoded string, it should be at the same address.